src/thunderbots/software/ai/hl/stp/play/kickoff_enemy_play.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ai/hl/stp/play/kickoff_enemy_play.h"
+
+#include "ai/hl/stp/evaluation/enemy_threat.h"
+#include "ai/hl/stp/evaluation/possession.h"
+#include "ai/hl/stp/play/play_factory.h"
+#include "ai/hl/stp/tactic/move_tactic.h"
+#include "ai/hl/stp/tactic/shadow_kickoff_tactic.h"
+#include "shared/constants.h"
+
+const std::string KickoffEnemyPlay::name = "KickoffEnemy Play";
+
+std::string KickoffEnemyPlay::getName() const
+{
+    return KickoffEnemyPlay::name;
+}
+
+bool KickoffEnemyPlay::isApplicable(const World &world) const
+{
+    return world.gameState().isTheirKickoff();
+}
+
+bool KickoffEnemyPlay::invariantHolds(const World &world) const
+{
+    return world.gameState().isTheirKickoff();
```

Comment:
suggest changing to something like `!isPlaying` (pseudocode) so that we finish this once normal play starts (and can transition to something like a `GrabBall` play).

Reply:
fixed